packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -237,6 +237,7 @@ module.exports = {
             name: 'stats',
             usage: '--stats <value>',
             type: String,
```

Comment:
Will be great to allow setup it like `type: [String, Boolean]`, what do you think, it is possible do on args-parser?

Reply:
Yes we can do this.

Reply:
Will be great :+1: